{"name": "Получение задачи с невалидным ID", "status": "passed", "steps": [{"name": "Генерируем данные для новой задачи", "status": "passed", "attachments": [{"name": "Сгенерированные данные", "source": "06863e70-af31-46c2-b4e1-0605ef362ea1-attachment.json", "type": "application/json"}], "start": 1746535615104, "stop": 1746535615105}, {"name": "Создаем задачу через API", "status": "passed", "attachments": [{"name": "Тело запроса", "source": "19b68e8d-6827-44a3-afa8-85da5e4744df-attachment.json", "type": "application/json"}, {"name": "Ответ сервера", "source": "6711de70-4f16-45ee-a680-60d5a69b28fa-attachment.json", "type": "application/json"}], "start": 1746535615105, "stop": 1746535615962}, {"name": "Отправляем GET запрос с некорректным ID: None", "status": "passed", "start": 1746535615962, "stop": 1746535617657}, {"name": "Проверяем код ответа и сообщение об ошибке", "status": "passed", "start": 1746535617657, "stop": 1746535617657}], "parameters": [{"name": "wrong_id", "value": "None"}], "start": 1746535615104, "stop": 1746535617657, "uuid": "34824694-8dd0-4bcf-9379-78b22b8bd71a", "historyId": "b8600f740e938b355dbf1259d889bd05", "testCaseId": "3e916150ead964e7afe384aca8020f86", "fullName": "tests.api.test_tasks.TestTasks#test_get_task_invalid", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Egors-MacBook-Air.local"}, {"name": "thread", "value": "21580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}